### PR TITLE
11.0 [IMP] Resolved issue of transalation.

### PR DESCRIPTION
**Pull request by AnuKPatel**
Improved By @Freni-SerpentCS

### CODE REVIEW

File: school_hostel/models/hostel.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def _compute_check_availability(self):
     student_ids = fields.One2many('hostel.student', 'room_id',
                                   string="Students")
 
-    _sql_constraints = [('room_no_unique', 'unique(room_no)',
+    _sql_constraints = [('room_no_unique', 'unique(room_no,name)',
```
Comment (JayVora-SerpentCS) [inline school_hostel/models/hostel.py]: make a good pep-8

**Comment by JayVora-SerpentCS** [comment]
@Freni-SerpentCS I already told you all files where you should change.
@AnuKPatel should do a simple https://github.com/JayVora-SerpentCS/OdooEduERP/search?q=week_day&unscoped_q=week_day  and see where we can change. its easy.

**Comment by AnuKPatel** [comment]
ok Let me do the needful. @JayVora-SerpentCS cc @Freni-SerpentCS

File: school_hostel/models/hostel.py (hunk 1 above)
Comment (AnuKPatel) [reply school_hostel/models/hostel.py]: Done.

**Comment by JayVora-SerpentCS** [comment]
> @Freni-SerpentCS I already told you all files where you should change.
> @AnuKPatel should do a simple https://github.com/JayVora-SerpentCS/OdooEduERP/search?q=week_day&unscoped_q=week_day and see where we can change. its easy.

this needs a look too. @Freni-SerpentCS knows. she will do once free.

**Comment by Freni-SerpentCS** [comment]
Done. @JayVora-SerpentCS 
CC: @AnuKPatel

**Comment by JayVora-SerpentCS** [comment]
> > @Freni-SerpentCS I already told you all files where you should change.
> > @AnuKPatel should do a simple https://github.com/JayVora-SerpentCS/OdooEduERP/search?q=week_day&unscoped_q=week_day and see where we can change. its easy.
> 
> this needs a look too. @Freni-SerpentCS knows. she will do once free.

@Freni-SerpentCS did you check here?

**Pull request merged**